README.md

```diff
@@ -128,18 +128,22 @@ make test
 
 Compiling CX on Windows requires a recent version of Git to be installed. 
 
-Before compiling CX, install dependencies with `pacman` (download MSYS2):
+Pacman is a utility which manages software packages.   
 
-```
-pacman -Sy
+To install pacman, download [Mysys2](https://www.msys2.org) and run the installer.
+
+When the installation is complete, click ` Run MSYS2 now `.     
+
+If MSYS2 has already been installed, run it through the start menu.
 
-pacman -S mingw-w64-x86_64-openal
+You can run a full system upgrade and install required dependencies with: 
+
+```
+pacman -Syu git mingw-w64-x86_64-openal base-devel mingw-w64-x86_64-toolchain
 
 if [ ! -a /mingw64/lib/libOpenAL32.a ]; then ln -s /mingw64/lib/libopenal.a /mingw64/lib/libOpenAL32.a; fi
 
 if [ ! -a /mingw64/lib/libOpenAL32.dll.a ]; then ln -s /mingw64/lib/libopenal.dll.a /mingw64/lib/libOpenAL32.dll.a; fi
-
-pacman -S --needed base-devel mingw-w64-x86_64-toolchain
 ```
 
```

Comment:
One more thing is needed. When installation is completed. PATH to `%MSYS2%/usr/bin` and `%MSYS2%/mingw64/bin` must be added to `Environment Variables` on Windows for `git` and `-gcc` to work.

Reply:
@archerixx let me know how to do that please.

Reply:
Press `WIN`+`R` and then type `SystemPropertiesAdvanced`. Select `Environment Variables`. Under `System Variables` select `Path` then click on `Edit`.
Click on `New` and add these two paths:
`C:\msys64\mingw64\bin`
`C:\msys64\usr\bin`
Save and Exit.

Restart any open Shells for changes to apply.